esp-ieee802154-examples/examples/receive_all_frames.rs

```diff
@@ -48,7 +48,12 @@ fn main() -> ! {
     wdt1.disable();
 
     println!("Start");
-    let mut ieee802154 = Ieee802154::new(&mut system.radio_clock_control);
+
+    #[cfg(feature = "esp32c6")]
+    let (_, _, radio) = peripherals.RADIO.split();
+    #[cfg(feature = "esp32h2")]
+    let (_, radio) = peripherals.RADIO.split();
```

Comment:
```suggestion
    let (.., radio) = peripherals.RADIO.split();
```

This can be simplified, same goes for the other examples.

Reply:
ah yes ... I keep forgetting about this syntax 👍